python/lsst/meas/algorithms/readTextCatalogTask.py

```diff
@@ -54,6 +54,12 @@ class ReadTextCatalogConfig(pexConfig.Config):
         doc=("Format of files to read, from the astropy.table I/O list here:"
              "http://docs.astropy.org/en/stable/io/unified.html#built-in-table-readers-writers")
     )
+    fill_values = pexConfig.ListField(
+        dtype=str,
+        default=[],
+        doc=("A list giving [<match_string>, '0', <optional column name>], which is used to mask"
```

Comment:
What is the `'0'` for here? It looks like that's the value to fill with; since there's an "optional column name", can't we have this be per-column, and thus use `NaN`?

Reply:
This is what is recommended by astropy (https://docs.astropy.org/en/stable/io/ascii/read.html#f1), but not actually required. If the `ConfigDictField` works out, I'll change this.